src/beyond_bets/transforms/market_analyser.py

```diff
@@ -0,0 +1,76 @@
+from pyspark.sql import DataFrame, SparkSession
+from beyond_bets.base.transform import Transform
+from beyond_bets.datasets.bets import Bets
+from pyspark.sql import functions as fn
```

Comment:
Rather import the exact functions needed

`from pyspark.sql.functions import col, sum`